### PR TITLE
Test comment 3

### PR DESCRIPTION
**Pull request by agustin-mrtz**

### CODE REVIEW

**Comment by agustin-mrtz** [comment]
Adding a comment :D to test 4178

**Comment by agustin-mrtz** [comment]
Posting another comment

**Comment by agustin-mrtz** [comment]
Comment 3

File: .lift/failing_tool.py [hunk 1]
```diff
@@ -1 +1,41 @@
...
+
+
+def emit_applicable():
+    print("true")
+
+def run():
+    print('[ { \"type\" : \"lift test dev1\" , \"message\" : \"testing lift in dev1\", \"file\" : \".lift/failing_tool.py\" , \"line\" : 1 } , \n { \"type\" : \"lift test dev1\" , \"message\" : \"testing lift in dev1\", \"file\" : \".lift/failing_tool.py\" , \"line\" : 2 } ]')
+    # raise Exception('Feeling under the weather')
+
+def main():
```
Comment (agustin-mrtz) [inline .lift/failing_tool.py:19]: hello testing 4178

**Comment by agustin-mrtz** [comment]
adding a comment for the 4th time

File: .lift/failing_tool.py [hunk 2]
```diff
@@ -1 +1,41 @@
...
+    parser = argparse.ArgumentParser(description='Failing Tool')
+    parser.add_argument('path', metavar='PATH', help='Path to code')
+    parser.add_argument('commit_hash', metavar='HASH', help='Commit hash')
+    parser.add_argument('command', metavar='COMMAND', help='Command')
+
+    args = parser.parse_args()
+
+    path = args.path
```
Comment (agustin-mrtz) [inline .lift/failing_tool.py:27]: bkjbkbk

File: .lift/failing_tool.py [hunk 3]
```diff
@@ -1 +1,41 @@
...
+
+    command = args.command
+
+    if command == "version":
+        emit_version()
+    elif command == "name":
+        emit_name()
```
Comment (hth313) [inline .lift/failing_tool.py:34]: So I put a comment here.

File: .lift/failing_tool.py [hunk 4]
```diff
@@ -1 +1,41 @@
...
+
+    args = parser.parse_args()
+
+    path = args.path
+
+    command = args.command
+
+    if command == "version":
+        emit_version()
+    elif command == "name":
```
Comment (agustin-mrtz) [inline .lift/failing_tool.py:33]: kbjkbk

File: .lift/config.toml [hunk 5]
```diff
@@ -0,0 +1 @@
+customTools = [".lift/failing_tool.py"]
```
[the remark this answers is not in the document]
Comment (agustin-mrtz) [reply .lift/config.toml:1]: eediting review comment

**Review by hth313 — COMMENTED** [review]
OK that was good. Some addition,

**Review by agustin-mrtz — COMMENTED** [review]
eedit reviews

File: src/main/kotlin/com/sonatype/test/errors/Foo.kt [hunk 6]
```diff
@@ -0,0 +1,18 @@
+package com.sonatype.test.errors
+
+import java.io.IOException
+
+class Foo {
```
Comment (agustin-mrtz) [inline src/main/kotlin/com/sonatype/test/errors/Foo.kt:5]: adding another

**Pull request closed without merging**